package.json

```diff
@@ -40,6 +42,7 @@
     "chai": "^3.5.0",
     "eslint": "^3.9.1",
     "eslint-config-screwdriver": "^2.0.9",
+    "eslint-plugin-import": "^2.9.0",
```

Comment:
I think if you update `eslint` and `eslint-config-screwdriver` versions, you don't need `eslint-plugin-import` (https://github.com/screwdriver-cd/scm-github/blob/master/package.json#L35-L36)